packages/platform/atoms/booker/BookerPlatformWrapper.tsx

```diff
@@ -88,6 +92,14 @@ export const BookerPlatformWrapper = (props: BookerPlatformWrapperAtomProps) =>
       setOrg(null);
       setSelectedMonth(null);
       setSelectedDuration(null);
+      if (props.rescheduleUid) {
+        // clean booking data from cache
+        queryClient.removeQueries({
+          queryKey: [BOOKING_RESCHEDULE_KEY, props.rescheduleUid],
+          exact: true,
+        });
+        setBookingData(null);
```

Comment:
we reset the booking data, previously the platform booker was remounted with the data from the previous booking while the fetch was in progress